tests/integration/targets/vmware_ops_vcenter_host_connection_test/tasks/main.yml

```diff
@@ -0,0 +1,53 @@
+---
+- name: Test
+  block:
+    - name: Add Resource Pool
+      ansible.builtin.include_tasks: call_resource_pool_role.yml
```

Comment:
technically, the `call_resource_pool_role.yml` is calling a module, not a role. Maybe better rename it to `manage_resource_pool.yml`?

Reply:
done